lisa/sut_orchestrator/azure/tools.py

```diff
@@ -364,3 +374,28 @@ def enable_ssh_on_windows(
             f"-CommandId 'RunPowerShellScript' "
             f"-Parameter @{{'PublicKey'='{public_key_data}'}}"
         )
+
+
+class Rpm(Tool):
```

Comment:
move it to base_tools folder, so it can be used by operating system in future. Refer to other base_tools. And I think the rpm is not installable for all distros, is it right?

https://github.com/microsoft/lisa/tree/main/lisa/base_tools

Reply:
Remember to refer it in tools folder like others in base_tools.